instance/templates/instance/ansible/secret-keys.yml

```diff
@@ -14,9 +15,14 @@ NOTIFIER_LMS_SECRET_KEY: '{{ secret_generator.NOTIFIER_LMS_SECRET_KEY }}'
 PROGRAMS_SECRET_KEY: '{{ secret_generator.PROGRAMS_SECRET_KEY }}'
 ECOMMERCE_SOCIAL_AUTH_EDX_OIDC_KEY : '{{ secret_generator.ECOMMERCE_SOCIAL_AUTH_EDX_OIDC_KEY }}'
 ECOMMERCE_SOCIAL_AUTH_EDX_OIDC_SECRET : '{{ secret_generator.ECOMMERCE_SOCIAL_AUTH_EDX_OIDC_SECRET }}'
-ECOMMERCE_EDX_API_KEY: '{{ secret_generator.EDXAPP_EDX_API_KEY }}' # Same as the edxapp API key
 EDXAPP_ANALYTICS_API_KEY: '{{ secret_generator.EDXAPP_ANALYTICS_API_KEY }}'
 FORUM_API_KEY: '{{ secret_generator.FORUM_API_KEY }}'
-EDXAPP_COMMENTS_SERVICE_KEY: '{{ secret_generator.FORUM_API_KEY }}'
-NOTIFIER_COMMENT_SERVICE_API_KEY: '{{ secret_generator.NOTIFIER_COMMENT_SERVICE_API_KEY }}'
-NOTIFIER_USER_SERVICE_API_KEY: '{{ secret_generator.NOTIFIER_USER_SERVICE_API_KEY }}'
+
+# Keys that need to match other keys.  The variables will be filled in by Jinja2 template
+# processing in Ansible rather than by the Django template processor, hence the verbatim block.
+{% verbatim %}
+ECOMMERCE_EDX_API_KEY: '{{ EDXAPP_EDX_API_KEY }}'
```

Comment:
@antoviaque In the old version, it was very hard to spot that some of the values are fundamentally different from the others:  They aren't individually generated keys, but are rather set to match other values.  The new version makes this intent much clearer than the old one by explicitly configuring them to match.  If you want two values to have the same computed value, you generally would not write

    a = f(x)
    b = f(x)

but rather

    a = f(x)
    b = a

to make clear that `a` and `b` are supposed to have the same value.  This change is made in the same spirit.